text/0000-rfc-process.md

```diff
@@ -0,0 +1,233 @@
+
+#### Stewardship
+
+The steward and their team should:
+ * work with the author of the RFC to ensure that the RFC is complete and implementable contingent upon approval. This can happen synchronously at team working groups or asynchronously via github and slack.
+ * raise visibility to and solicit feedback from other stakeholders including the TOC, other teams, and the community at large. This can happen synchronously at the project working group and asynchronously via github and slack.
+ * drive consensus for the RFC by incorporating feedback from stakeholders so that the RFC has the best possible chance of approval during the voting process.
+ * ensure there is a plan in place to implement the RFC in a reasonable time frame, contingent upon approval. The team itself need not implement the RFC but we should not approve RFCs for which we have no concrete plan to implement.
+ * in the case of complicated or risky RFCs, a POC should be developed at this stage to validate and de-risk the proposed design.
+
+#### Voting
+
+Ideally the voting should be a formality and not a moment to discover new disagreement, consensus already haven been driven by the steward.
+
+When the steward deems the RFC ready and likely to be accepted they should formally call for votes and set an end date for voting. This process does not prescribe a length for the voting window, but stewards should make a good faith effort to ensure that all interested parties, and in particular those with binding votes, have adequate opportunity to review the finalized RFC and cast votes.
+
+The RFC may not be edited during the voting window.
+
+For project RFCs, all TOC members and all team leads are given a binding vote.
+
+Additionally, for team RFCs, all team maintainers are given a binding vote.
+
+Votes are cast via reviews on the RFC PR. Accepting the PR signifies a yes vote while a request for changes signifies a no vote. If all members with a binding vote vote in the affirmative, the voting window may close early.
+
+##### Acceptance
+
+If the end date of the vote arrives without a no vote from a member with binding vote, the RFC is accepted. It will be merged into the repo and assigned a number. The steward should create a tracking issue to coordinate implementation and link the tracking issue in the RFC header. When this is complete, implementation can begin.
+
+##### Rejection
+
+If a single no vote from a member with binding vote is cast before the arrival of the end date the RFC is immediately rejected and the PR should be closed.
+
+The same RFC may be re-opened and brought to a vote again in the future assuming the concerns that lead to the no vote are addressed.
+
+##### Implementation
```

Comment:
Minor: It seems like this header is either a) of the wrong level or b) empty.

Reply:
@ekcasey any comment on this?

Reply:
```suggestion
##### Implementation

After the RFC has been accepted, any team or individual may begin implementing the changes it defines. 
```